### PR TITLE
Merge develop to create release 1.0

### PR DESCRIPTION
**Pull request by glormph**

### CODE REVIEW

**Comment by glormph** [comment]
Merge only after last test of full pipeline on whole bam file has been run by @viklund

**Review by viklund — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -87,15 +101,20 @@ usage message for information on them.
...
 Finally the `runtime` section has the different runtimes for the different
 parts of the workflow. `fermikit` has it's own timespec since that is a very
 long running program, otherwise the workflow differentiates between `callers`
 and other supporting `simple` single-core jobs.
 
 
+## Support
+
+If you need help with this module, please create a [support issue](https://github.com/NBISweden/wgs-structvar/issues/new?title=Support&body=Describe%20your%20error%20here.%0A%0AInclude%20as%20much%20information%20as%20possible,%20especially%20what%20system%20you%20ran%20this%20on,%20the%20.nextflow.log%20and%20the%20terminal%20output%20from%20the%20program.%0A%0ARemember%20that%20this%20is%20a%20public%20form,%20don%27t%20include%20private%20information.) in github.
+
```
Comment (andkaha) [inline README.md:117]: Add shortened link as alternative?

File: main.nf [hunk 2]
```diff
@@ -360,6 +348,12 @@ process variant_effect_predictor {
               exit 1;;
     esac
 
+    # VEP failes files without variants, but the pipeline should still run
```
Comment (andkaha) [inline main.nf:351]: Nitpick: Shell variables should be double quoted. `"\$INFILE"`, etc.

Lines 353--374

**Review by andkaha — COMMENTED** [review]
Suggestions:

* Give alternative shortened link.
* Double quote all shell variables.

**Pull request merged**